README.md

```diff
@@ -28,31 +28,134 @@ flag is highly recommended for those new to [pip](https://pip.pypa.io).
 A PEX executable (Windows not supported) and source releases are
 [here](https://github.com/planetlabs/planet-client-python/releases/latest).
 
-## Documentation
+## API Key
 
-Online documentation:
-https://planetlabs.github.io/planet-client-python/index.html
+The planet API requires an account for use. [Signup here](https://www.planet.com/explorer/?signup).
 
-Documentation is also provided for download
-[here](https://github.com/planetlabs/planet-client-python/releases/latest).
+This can be provided via the environment variable `PL_API_KEY` or the flag `-k` or `--api-key`.
 
+Using `planet init` your account credentials (login/password) can be used to obtain the api key.
 
-## Development
+## Quick Start
 
-To contribute or develop with this library, see
-[CONTRIBUTING](https://github.com/planetlabs/planet-client-python/CONTRIBUTING.md)
+### Python Library
 
+The python library is asynchronous, which greatly speeds up many interactions
+with the planet API. 
 
-## API Key
+```python
+import asyncio
+import os
+from planet.api import APlanetSession, AOrdersClient
+from planet.api.order_details import OrderDetails, Product
```

Comment:
It seems likely that the name `Product` would be a good candidate for name collisions. I'd suggest namespacing this instead of importing the class directly.

Reply:
good point re: conflicts. A nice usage would be to import this directly from `planet` so maybe a name change to `OrderProduct` (and `OrderTool` etc) would make sense.

Reply:
or maybe keep this namespaced. I just don't really like the namespace `order_details`

Reply:
Those look really nice from a user perspective. I also like removing the redundant `Planet` from `planet.ASession`. I'll go ahead and implement this change.